identity/webapp/pages/account/validated.tsx

```diff
@@ -16,13 +16,14 @@ type Props = {
 };
 
 const ValidatedPage: NextPage<Props> = ({ success, message }) => {
+  const urlUsed = message === 'This URL can be used only once';
```

Comment:
Maybe put a comment linking to https://auth0.com/docs/brand-and-customize/email/email-template-descriptions#redirect-to-results-for-verification-email-template as this is indeed weird but necessary